.github/workflows/main.yml

```diff
@@ -0,0 +1,73 @@
+name: CI
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+  push:
+    branches:
+      - master
+
+jobs:
+  ci:
+    strategy:
+      fail-fast: true
+
+    name: CI (stable)
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install Rust Stable
+        run: |
+          rustc -vV
+          rustup update stable
+          rustup default stable
+          rustc -vV
+
+      - name: test (netlink-sys)
+        run: |
+          cd netlink-sys
+          cargo test
+          cargo test --features tokio_socket
```

Comment:
Not necessarily to be addressed in this PR but `cargo test --features smol_socket` currently fails. We should fix this and add it to CI afterward.